app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,21 @@
 <resources>
     <string name="app_name">FurryFriendFinder</string>
+    <string name="sex">Sexo</string>
+    <string name="kind">Especie</string>
+    <string name="place">Lugar</string>
+    <string name="sterilized">Esterilizado</string>
+    <string name="search">Buscar</string>
+    <string name="dangerous">Peligroso</string>
+    <string name="breed">Raza</string>
+
+    <string-array name="sexOptions">
+        <item name="male">Macho</item>
+        <item name="female">Hembra</item>
+        <item name="indefinite">Indefinido</item>
+    </string-array>
+
+    <string-array name="answerYesNo">
+        <item name="yes">Si</item>
+        <item name="no">No</item>
+    </string-array>
```

Comment:
En los strings, recomiendo añadir la parte en que se están utilizando, me explico, si estamos en la parte de búsqueda, poner screen_input_title_sex, o algo así, s una sugerencia, en este caso como la app es pequeña, no afecta mucho.

Reply:
eso como queráis, si hace falta lo cambio, sin problema

Reply:
Estoy de acuerdo con Joel, aunque sea una App pequeña creo que es una buena practica que facilita la lectura.

Reply:
solo como comentario, que si lo hacemos así seguramente tendremos que duplicar valores, porque si en dos pantallas distintas se usa el mismo, con esa notación tendríamos que tener dos, por ejemplo, sexo, tendríamos que tener screen_input_title_sex y screen_detail_title_sex las dos con el mismo valor, lo que si que es cierto, que se puede detallar un poco más, por ejemplo en vez de <string name="sex">Sexo</string> <string name="sex_label">Sexo</string> o algo así, para saber que es algo para usar como una etiqueta sin limitarlo a una pantalla, como lo veis?